linters/.jshintrc

```diff
@@ -1,26 +1,13 @@
 {
-  "globals": {
```

Comment:
Adding both `browser` and `node` environments eliminates the need for this list.